pkg/cmd/triggerbinding/testdata/TestTriggerBindingDescribe_WithParams.golden

```diff
@@ -0,0 +1,7 @@
+Name:        tb1
+Namespace:   ns
+
+Params
+
+ NAME   VALUE
+ key    value
```

Comment:
It should be in bullet points

Reply:
I believe it's set up so decorations are removed from unit testing.

Reply:
Yep
Bullet points are removed from unit testing

Output of tkn triggerbinding describe  as follows
```
Name:        pipeline-binding
Namespace:   default

⚓ Params

 NAME                 VALUE
 ∙ gitrevision        master
 ∙ gitrepositoryurl   $(body.repository.url)
 ∙ contenttype        $(header.Content-Type)
```

Reply:
ohh ohk, thanks